Dockerfile

```diff
@@ -1,31 +1,32 @@
-FROM ubuntu:14.04
+FROM onsdigital/docker-aws-apache-wsgi:python-3.4.2
 
-ENV RUNTIME_PACKAGES="python3 postgresql python-psycopg2"
-ENV BUILD_PACKAGES="curl build-essential python3-dev libpq-dev ca-certificates libffi-dev libssl-dev git"
+# Install eq-survey-runner Python packages
+WORKDIR /tmp/eq-survey-runner
+ADD requirements.txt /tmp/eq-survey-runner/requirements.txt
```

Comment:
It improves the caching and build times after the 1st build. but i believe this is going anyway.